docs/frontend/in-depth/axios.md

```diff
@@ -0,0 +1,78 @@
+# API requests power by axios
+
+All API requests are supported by the [nuxt/axios][axios] library,<br>
+which provides us with full support for API queries.
+
+### Introduction
+---
+The main configuration of the library is located in the file `nuxt.config.js`, <br>
+there are the most important things needed to run the library on our system. <br>
+
+To configure all necessary settings for API queries, <br>
+overwrite configurations by creating `axios.js` plugin in  **Core** module.<br>
```

Comment:
you need to overwrite configurations by creating `axios.js` plugin in  **Core** module.<br>